projects/Mallard/src/helpers/push-download-failsafe.ts

```diff
@@ -0,0 +1,39 @@
+import BackgroundFetch from 'react-native-background-fetch'
+import { pushTracking } from './push-tracking'
+import { clearAndDownloadIssue } from './clear-download-issue'
+
+const pushDownloadFailsafe = () => {
+    BackgroundFetch.configure(
+        {
+            minimumFetchInterval: 120, // Every 2 hours
```

Comment:
Why every two hours? Can this be longer if it's likely to effect battery life of device?

Reply:
As suggested in the comment above, it's an initial test, we can be smarter after that. This PR can be not merged until that smarter way is implemented.

Reply:
Further look into this, this is a minimum interval, so this will actually trigger based on how much you use the app. The more you use it, the closer to 2 hours it will be, the less, then it will be longer.

It may be worth making this happen at a faster interval to catch more.